doc/guides/debugging/probes.rst

```diff
@@ -0,0 +1,143 @@
+.. _debug-probes:
+
+Debug Probes
+############
+
+Many supported boards in Zephyr include a second microcontroller that serves as
```

Comment:
Maybe it's because I work for an IC company, but when I see "integrated" like this I think integrated into the SoC. I want to emphasize the fact that there is a second MCU and firmware associated it (and maybe someday it will run zephyr too!)

Reply:
Heh, fair. How about "on-board"?